app/src/main/res/layout/filter_org_unit.xml

```diff
@@ -110,7 +109,7 @@
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="?colorPrimaryDark"
+            android:background="#32000000"
```

Comment:
Can we set this color in the colors.xml file. I see it is set on different xml files.